### PR TITLE
Add more events to smb2-com-set-info

### PR DESCRIPTION
**Pull request by mauropalumbo75**

### CODE REVIEW

File: src/analyzer/protocol/smb/smb2-com-set-info.pac [hunk 1]
```diff
@@ -81,13 +253,120 @@ type SMB2_file_disposition_info(sir: SMB2_set_info_request) = record {
...
+
+type SMB2_file_pipe_info(sir: SMB2_set_info_request) = record {
+	read_mode 	: uint32;
+	completion_mode : uint32;
+} &let {
+	proc: bool = $context.connection.proc_smb2_set_info_request_file_pipe(this);
+};
+
+type SMB2_file_position_info(sir: SMB2_set_info_request) = record {
+	current_byte_offset 	: int64;
```
Comment (sethhall) [inline src/analyzer/protocol/smb/smb2-com-set-info.pac]: I see you're using ints (as opposed to uints) in a lot of places here.  Should those really be ints?  It seems that a negative value for current_byte_offset would be odd.

Comment (sethhall) [reply src/analyzer/protocol/smb/smb2-com-set-info.pac]: Well, I shouldn't have said in "a lot" of places.  I guess you're only using them in a few places.

Comment (mauropalumbo75) [reply src/analyzer/protocol/smb/smb2-com-set-info.pac]: Some fields are signed integer as in the specs given in
[MS-FSCC] - v20180912
File System Control Codes
Copyright © 2018 Microsoft Corporation
Release: September 12, 2018

**Comment by sethhall** [comment]
Ah, of course.  I should have look at the spec. Not sure why they're using signed ints for some of these fields, but whatever.  Thanks!

**Comment by mauropalumbo75** [comment]
Honestly, I am not sure why some of these fields are signed rather than unsigned ints in Microsoft's specifications, but I thought better to do as they say...

**Pull request merged**